src/dmd/module.h

```diff
@@ -53,6 +53,13 @@ class Package : public ScopeDsymbol
     Module *isPackageMod();
 };
 
+struct QualifiedID
+{
+    Identifiers *packages;
+    Identifier *ident;
+    const char *toChars();
```

Comment:
`const(char)* toChars() const;`

Reply:
I think you meant `const char *toChars() const` ?